### DISCUSSION TITLE
Problem running in 1.20.6

### DISCUSSION THREAD

**Discussion by bjnorton80** [GeorgH93/Minepacks#316] [General]
Is anyone else having issues with plugin running 1.20.6? Backpack is just placed on the ground. It wont open. Also, the minepack plugin is listed in red when I check loaded plugins on my server.

**Comment by GeorgH93** [comment]
Please update the plugin to the latest version.

**Reply by bjnorton80** [reply]
I updated to 2.4.27

**Reply by GeorgH93** [reply]
The latest version is [2.4.28](https://github.com/GeorgH93/Minepacks/releases/download/v2.4.28/Minepacks-2.4.28-Release.jar).

**Reply by bjnorton80** [reply]
I have totally remover the plugin and the reinstalled. download latest .jar file and it still doesn't work. I cannot access the commands in game, /backpack. When I check my plugins in game with "/plugins", minepacks is red and all others are green.

**Reply by bjnorton80** [reply]
This is what my console reports.

13.05 09:49:57 [Server] INFO Minepacks has been disabled.  :( 
13.05 09:49:56 [Server] WARN The updater could not find any files for the project id 83445 
13.05 09:49:56 [Multicraft] Skipped 11 lines due to rate limit (30/s)
13.05 09:49:56 [Server] INFO at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:682) ~[paper-1.20.5.jar:git-Paper-22]
13.05 09:49:56 [Server] INFO at org.bukkit.craftbukkit.CraftServer.enablePlugins(CraftServer.java:551) ~[paper-1.20.5.jar:git-Paper-22]
13.05 09:49:56 [Server] INFO at org.bukkit.craftbukkit.CraftServer.enablePlugin(CraftServer.java:639) ~[paper-1.20.5.jar:git-Paper-22]
13.05 09:49:56 [Server] INFO at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:519) ~[paper-api-1.20.5-R0.1-SNAPSHOT.jar:?]
13.05 09:49:56 [Server] INFO at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.enablePlugin(PaperPluginManagerImpl.java:109) ~[paper-1.20.5.jar:git-Paper-22]
13.05 09:49:56 [Server] INFO at io.papermc.paper.plugin.manager.PaperPluginInstanceManager.enablePlugin(PaperPluginInstanceManager.java:202) ~[paper-1.20.5.jar:git-Paper-22]
13.05 09:49:56 [Server] INFO at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:287) ~[paper-api-1.20.5-R0.1-SNAPSHOT.jar:?]
13.05 09:49:56 [Server] INFO at Minepacks.jar/at.pcgamingfreaks.MinepacksStandalone.Bukkit.Minepacks.onEnable(Minepacks.java:118) ~[Minepacks.jar:?]
13.05 09:49:56 [Server] INFO at Minepacks.jar/at.pcgamingfreaks.MinepacksStandalone.Bukkit.Database.Language.<init>(Language.java:37) ~[Minepacks.jar:?]
13.05 09:49:56 [Server] INFO at Minepacks.jar/at.pcgamingfreaks.MinepacksStandalone.libs.at.pcgamingfreaks.Bukkit.Config.Language.<init>(Language.java:36) ~[Minepacks.jar:?]
13.05 09:49:56 [Server] INFO at Minepacks.jar/at.pcgamingfreaks.MinepacksStandalone.libs.at.pcgamingfreaks.Config.LanguageWithMessageGetter.<init>(LanguageWithMessageGetter.java:38) ~[Minepacks.jar:?]
13.05 09:49:56 [Server] INFO at Minepacks.jar/at.pcgamingfreaks.MinepacksStandalone.libs.at.pcgamingfreaks.Config.ReflectionMessageFactory.<clinit>(ClassReflectionMessageFactory.java:46) ~[Minepacks.jar:?]
13.05 09:49:56 [Server] INFO at io.papermc.paper.pluginremap.reflect.PaperReflectionHolder.forName(Unknown Source) ~[paper-1.20.5.jar:git-Paper-22]
13.05 09:49:56 [Server] INFO at io.papermc.reflectionrewriter.runtime.AbstractDefaultRulesReflectionProxy.forName(AbstractDefaultRulesReflectionProxy.java:68) ~[reflection-rewriter-runtime-0.0.1.jar:?]
13.05 09:49:56 [Server] INFO at java.base/java.lang.Class.forName(Class.java:513) ~[?:?]
13.05 09:49:56 [Server] INFO at java.base/java.lang.Class.forName(Class.java:534) ~[?:?]
13.05 09:49:56 [Server] INFO at java.base/java.lang.Class.forName0(Native Method) ~[?:?]
13.05 09:49:56 [Server] INFO at Minepacks.jar/at.pcgamingfreaks.MinepacksStandalone.libs.at.pcgamingfreaks.Bukkit.Message.Sender.SendMethod.<clinit>(SendMethod.java:40) ~[Minepacks.jar:?]
13.05 09:49:56 [Server] INFO at Minepacks.jar/at.pcgamingfreaks.MinepacksStandalone.libs.at.pcgamingfreaks.Bukkit.Message.Sender.TitleSender.<clinit>(TitleSender.java:36) ~[Minepacks.jar:?]
13.05 09:49:56 [Server] INFO at Minepacks.jar/at.pcgamingfreaks.MinepacksStandalone.libs.at.pcgamingfreaks.Bukkit.Protocol.TitleMessagePacketFactory_1_20_R4.makeTitlePacket(TitleMessagePacketFactory_1_20_R4.java:40) ~[Minepacks.jar:?]
13.05 09:49:56 [Server] INFO at Minepacks.jar/at.pcgamingfreaks.MinepacksStandalone.libs.at.pcgamingfreaks.Bukkit.Util.Utils_1_20_R4.jsonToIChatComponent(Utils_1_20_R4.java:56) ~[Minepacks.jar:?]
13.05 09:49:56 [Server] INFO java.lang.NoClassDefFoundError: org/bukkit/craftbukkit/v1_20_R4/util/CraftChatMessage
13.05 09:49:56 [Server] ERROR Error occurred while enabling Minepacks v2.4.28-T20240424192353 (Is it up to date?)
13.05 09:49:56 [Server] INFO Config file successfully loaded.
13.05 09:49:56 [Server] INFO File "/config.yml" extracted successfully!
13.05 09:49:55 [Server] INFO Starting Minepacks in standalone mode!
13.05 09:49:55 [Server] INFO Enabling Minepacks v2.4.28-T20240424192353

**Reply by GeorgH93** [reply]
Looks like you are using paper. Since they decided to change stuff again, you will have to wait till I have time to add support for this.
You can use Spigot meanwhile.

**Comment by bjnorton80** [comment]
backpacks started working again. But now I have 3 backpacks and cant get rid of the extras. Also started having Steve Heads shop up when joining server and when traveling to the nether and back
